bin/git-theta

```diff
@@ -110,4 +122,5 @@ def track(args):
 
 if __name__ == "__main__":
     args = parse_args()
+    git_utils.set_hooks()
```

Comment:
Should setting hooks be part of the install command? Or is there a reason to run it everytime?

Reply:
Yeah, nice catch. It shouldn't be run as part of install since there isn't any repo to put the hooks into.